client/app/routes/create-account.tsx

```diff
@@ -84,6 +122,13 @@ export default function CreateAccount() {
 										{emailState.error.message}
 									</small>
 								)}
+								{!emailState?.error &&
+									actionData?.type === "conflict" &&
+									actionData?.field === "email" && (
+										<small className="text-red-500 pl-2">
+											{actionData.message}
+										</small>
+									)}
```

Comment:
You could have a variable up there called `emailError`

```js
...
const emailError = emailState?.error || (actionData?.type === 'conflict' && actionData.field == 'email' && actionData.message)
...
```

Then on line 120, you do 

```jsx
{emailError && (<small ...>{emailError}</small>
```

Then you can remove L125-131

What do you think?

Reply:
Yeah that is better.